README.md

```diff
@@ -802,66 +787,53 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   STATES = ['draft', 'open', 'closed']
   ~~~
 
-* Usage of trailing comma in multi-line collection literals is encouraged. It
-  makes diffs smaller and more meaningful.
+* Append a trailing comma in multi-line collection literals.
 
   ~~~ ruby
-  # not encouraged
+  # bad
   {
     foo: :bar,
     baz: :toto
   }
 
-  # encouraged
+  # good
   {
     foo: :bar,
     baz: :toto,
   }
   ~~~
 
-* Prefer the literal array syntax to `%i`.
-
-  ~~~ ruby
-  # bad
-  STATES = %i(draft open closed)
-
-  # good
-  STATES = [:draft, :open, :closed]
-  ~~~
-
 * When accessing the first or last element from an array, prefer `first` or
   `last` over `[0]` or `[-1]`.
 
-* Avoid the use of mutable objects as hash keys.
+* Avoid mutable objects as hash keys.
 
 * Use the Ruby 1.9 hash literal syntax when your hash keys are symbols.
 
-* Don't mix the Ruby 1.9 hash syntax with hash rockets in the same hash literal.
-  When you've got keys that are not symbols stick to the hash rockets syntax.
-
   ~~~ ruby
   # bad
-  { a: 1, 'b' => 2 }
+  { :a => 1, :b => 2 }
 
   # good
-  { :a => 1, 'b' => 2 }
+  { a: 1, b: 2 }
   ~~~
 
-* Use `Hash#key?` instead of `Hash#has_key?` and `Hash#value?` instead of
-  `Hash#has_value?`. As noted
-  [here](http://blade.nagaokaut.ac.jp/cgi-bin/scat.rb/ruby/ruby-core/43765) by
-  Matz, the longer forms are considered deprecated.
+* Avoid mixing the Ruby 1.9 hash syntax with hash rockets in the same hash
```

Comment:
We could probably merge this with the previous rule by saying something like this and combining the examples.

```markdown
* Use shorthand hash literal syntax when **all** keys are symbols
```

I also don't think referring to Ruby 1.9 is very clear, as most developers at Shopify will have started writing Ruby well into 2.x.